tests/test_core.py

```diff
@@ -8523,6 +8523,20 @@ def test_emscripten_async_call(self):
     self.set_setting('EXIT_RUNTIME')
     self.do_run_in_out_file_test(test_file('core/test_emscripten_async_call.c'))
 
+  @parameterized({
+    '': ([],),
+    'no_dynamic_execution': (['-s', 'DYNAMIC_EXECUTION=0'],)
```

Comment:
why is DYNAMIC_EXECUTION relevant here?

Reply:
Because as you can see from changes to Embind, it has two different paths that depend on DYNAMIC_EXECUTION=0/1.

One is walking through types each time, another constructs a Function from string at runtime once and then reuses it, I assume for speed-up.

I hope we won't have this in the new binding system as realistically those bindings should be just generated from types AOT, but for now it is what it is...

Reply:
I see, thanks. sgtm.